docs/cardano/liquid-staking/c1-overview/contract-overview.mdx

```diff
@@ -67,7 +67,9 @@ Since rewards are automatically compounding, you do not need to call this functi
 
 *sstMilkADA* is an ERC20 token
```

Comment:
I noticed you didn't change the documentation about the `sstMilkADA` token. Is this still accurate?

Reply:
I did update it - it's accurate right now.

Reply:
Just deleting `sstMilkADA` definitely isn't right because as we've discussed there are two contracts: an ADA-denominated contract and a share-denominated contract

Reply:
What is the name and symbol of both contracts ? ADA denominated contract which we have currently is called `StakedMilkAda` and its symbol is `stMilkAda` right ? Reason I chnaged `sstMilkADA` to `stMilkADA`. Also, I did update the `contract-overview` docs.

Reply:
You can read [this file](https://github.com/dcSpark/milkomeda-documentation/blob/liquid-staking/docs/cardano/liquid-staking/c1-overview/tokens.mdx) to learn about the difference between the two

Reply:
@SebastienGllmt though the `tokens` doc describes both tokens `sstMilkADA`  and `stMilkADA`, but the `contract-overview` docs talks more of the `stMilkADA` contract in more details; and not the `sstMilkADA`  contract. Also from the docs, I can infer that the current `StakedMilkAda` contract is the `stMilkADA` token, and the `sstMilkADA` is yet to be implemented.